src/db/models/leaderboard.ts

```diff
@@ -0,0 +1,50 @@
+import mongoose, { Document } from 'mongoose';
+
+export interface ITeamData {
+  team: string;
+  score: number;
+  attemptsLeft: number;
+  delta: string;
+}
+
+export interface ILeaderboard extends Document {
+  timestamp: Date;
+  data: ITeamData[];
+}
+
+const leaderboardSchema = new mongoose.Schema<ILeaderboard>(
+  {
+    timestamp: {
+      type: Date,
```

Comment:
This is actually a string, but I will convert it to a Date object in the server. So leave as is